### ISSUE TITLE
Fix pull request template's markdown, reduce the required content

### ISSUE THREAD

**Issue by tormi** [wunderio/drupal-project#385]
The current pull request template has too many elements and content. Let's try to reduce it to a reasonable level.

Also, there are several syntax issues with the pull request template markdown:

<img width="963" alt="Screenshot 2024-01-05 at 19 04 11" src="https://github.com/wunderio/drupal-project/assets/146800/674e545d-ed63-4883-be30-06df503062d5">

**Comment by tormi** [comment]
We should probably introduce the markdown validation check, too.

**Comment by sampoturve** [comment]
I wonder whether the "Best practices" section is something that should be rethinked (or removed even), as it currently only provides a summary for accessibility. There are basically million best practices one could list here, and like the accessibility things, they should already be required on the definition of done of everything.

**Comment by tormi** [comment]
I totally agree. PR template should be really minimal, precise and not an obstacle as it's now (at least for me)
.
